client/apps/Planning/PlanningListSubNav.tsx

```diff
@@ -22,14 +22,17 @@ interface IProps {
     currentInterval: 'DAY' | 'WEEK' | 'MONTH';
     sortOrder: SORT_ORDER;
     sortField: SORT_FIELD;
-
+    userList:Array<{label: string, onSelect(): void}>;
```

Comment:
This prop is never used, and can be removed (along with it's assignment inside the `mapStateToProps` function